hazelcast/src/main/java/com/hazelcast/internal/server/tcp/SingleProtocolEncoder.java

```diff
@@ -97,6 +99,7 @@ private boolean sendProtocol() {
     // Swap this encoder with the next one
     protected void setupNextEncoder() {
         channel.outboundPipeline().replace(this, outboundHandlers);
+        channel.outboundPipeline().wakeup();
```

Comment:
I think we should avoid the call when we're inside the outbound pipeline already. I.e. distinguish the two calls (within and outside the outbound pipeline).

Reply:
I removed those parts. Now this PR will only solve NPE issue. I will create a seperate PR for SymmetricEncryptionSmokeTest blacklist issue.